src/main/java/ch/uzh/ifi/seal/soprafs20/field/Field.java

```diff
@@ -27,26 +27,35 @@ public void addAdjacency(Field adjacentField){
         }
     }
 
+
     public void removeAdjacency(Field removeField) {
         if (this.adjacencyList.contains(removeField)) {
             this.adjacencyList.remove(removeField);
         }
     }
 
-    public List<Field> getAdjacentFields() {
+    public List<Field> getAdjacencyList() {
         return this.adjacencyList;
     }
 
+    public void setAdjacencyList(List<Field> adjacencyList) {
+        this.adjacencyList = adjacencyList;
+    }
+
     public Figure getOccupant() {
         return occupant;
     }
 
     public void setOccupant(Figure occupant){
-        return;
+        this.occupant = occupant;
     }
 
     public long getId(){return id;};
 
+    public void setId(long id) {
+        this.id = id;
+    }
+
```

Comment:
`id` should probably be automatically generated (if `id`is a key in the database)